devenv: add comment explicity usage around options defined

sahid opened the pull request:
The start.sh's script is defining two times CILIUM_OPT and
CILIUM_OPERATOR_OPT inside the sysctl services env script. This change
is adding small comment to explain usage. It also ensures that
everyone understand that this was intentional ;-)

aanm reviewed (changes requested):
>The current algorithm was overriding options by setting CILIUM_OPT for
kv store and without.

@sahid Thank you for the PR, however this is intentional since developers might want to quickly use one mode or the other without having to restart the development VM. To use one or the other the developer simply has to change the order of the `CILIUM_OPTS` in the `/etc/sysconfig/cilium` file.

sahid committed:
devenv: add comment explicity usage around options defined

The start.sh's script is defining two times CILIUM_OPT and
CILIUM_OPERATOR_OPT inside the sysctl services env script. This change
is adding small comment to explain usage. It also ensures that
everyone understand that this was intentional ;-)

Signed-off-by: Sahid Orentino Ferdjaoui <sahid.ferdjaoui@industrialdiscipline.com>
